app/models/user.rb

```diff
@@ -380,6 +380,10 @@ def organization_and_child_ids
     taxonomy_and_child_ids(:organizations)
   end
 
+  def my_taxonomies_ids
```

Comment:
They are not for this PR. I'm fine to extract them to a separate PR but I strongly want them to be defined and available. And this should be the only way we use to get user taxonomies. Do you want me to open refactoring issue for it?